helpers/user_agent_helper_test.go

```diff
@@ -0,0 +1,16 @@
+package helpers_test
+
+import (
+	"go-crawler-challenge/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("UserAgentHelper", func() {
+	Describe("#RandomUserAgent", func() {
+		userAgent := helpers.RandomUserAgent()
+
+		Expect(len(userAgent)).NotTo(BeZero())
```

Comment:
The tests are a bit light. We can probably test for:
- The presence of the common string "Mozilla/5.0"
- The user agent string when split based on the spaces must contain between 4 and 5 items:
![image](https://user-images.githubusercontent.com/696529/108967683-65da6580-76b2-11eb-99d0-12b080d158dc.png)

![image](https://user-images.githubusercontent.com/696529/108967804-89051500-76b2-11eb-9017-b8ab2f312d0d.png)

Reply:
I've checked this with regex in test already ✌️